dll/win32/kernel32/client/sysinfo.c

```diff
@@ -380,13 +380,27 @@ GetNumaProcessorNode(IN UCHAR Processor,
     return TRUE;
 }
 
-/*
- * @implemented
- */
+/**
+ *
+ * @name GetNumaAvailableMemoryNode
+ * @implemented NT 5.2
+ *
+ *     Retrieves the amount of memory available in the specified node.
```

Comment:
It's better after your changes, but I'd appreciate if the function and parameter descriptions were also changed at least slightly.